### PR TITLE
added persian team members.

### PR DESCRIPTION
**Pull request by sattarfeizollahibarough**
Hi, This PR is added regarding the following PR:
https://github.com/kubernetes/website/pull/22565

### CODE REVIEW

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> @sattarfeizollahibarough: The following test **failed**, say `/retest` to rerun all failed tests:
> Test name 	Commit 	Details 	Rerun command
> pull-org-test-all 	[72aa9e9](https://github.com/kubernetes/org/commit/72aa9e9c73361fec11466b3a55b76e323cadf119) 	[link](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/org/2051/pull-org-test-all/1285441883601899521) 	`/test pull-org-test-all`
> 
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes&repo=org&pr=2051). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3sattarfeizollahibarough). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes/org/issues?q=is:issue+is:open) when you hit one in your PR.

In the log file mentioned that config_test.go failed. I should add the members into org.yaml file too.

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough 👋 Thanks for your initiative. If I understand correctly, I think each team member needs to open their own [issue requesting membership in the kubernetes org](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E). 

In each issue:
1. Mention that you're starting a new localization for Persian
2. Assign a [SIG Docs chair or tech lead](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) for approval `/assign <username>`

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> @sattarfeizollahibarough 👋 Thanks for your initiative. If I understand correctly, I think each team member needs to open their own [issue requesting membership in the Kubernetes org](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E).
> 
> In each issue:
> 
>     1. Mention that you're starting a new localization for Persian
> 
>     2. Assign a [SIG Docs chair or tech lead](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) for approval `/assign <username>`

@zacharysarah , I have checked the Docs and it is mentioned that we should submit the request, However, our team does not have any sponsor from current members of the Kubernetes community 😕. And I also have a tremendous drive to contribute to the Kubernetes Docs.

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough I’m willing to sponsor you. Membership for new localizations is a well-established precedent.

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> @sattarfeizollahibarough I’m willing to sponsor you. Membership for new localizations is a well-established precedent.

@zacharysarah It's very kind of you to offer this and made me happy 😃 . As far as, it is mentioned in the submission form I need two sponsors. Is there someone else to offer such a generous offer?

**Comment by zacharysarah** [comment]
> Is there someone else to offer such a generous offer?

@sftim @jimangel @celestehorgan 👋

**Comment by sftim** [comment]
I'm happy to sponsor as well.

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> I'm happy to sponsor as well.

@sftim I really appreciate your help. So, I will submit my request.

**Comment by kbhawkey** [comment]
Is there remaining work to be done?

**Comment by nikhita** [comment]
:wave: is this PR good to go?

**Review by onlydole — APPROVED** [review]
Thank you, @sattarfeizollahibarough! This looks good to me 👍🏼

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -311,6 +311,7 @@ members:
 - fabianvf
 - fabriziopandini
 - fabxc
+- FaezeAzimi
```
Comment (spiffxp) [inline config/kubernetes/org.yaml:314]:
```suggestion
```

File: config/kubernetes/org.yaml [hunk 2]
```diff
@@ -537,6 +538,7 @@ members:
 - KashifSaadat
 - Kashomon
 - kaslin
+- kasramp
```
Comment (spiffxp) [inline config/kubernetes/org.yaml:541]:
```suggestion
```

File: config/kubernetes/org.yaml [hunk 3]
```diff
@@ -883,6 +885,7 @@ members:
 - sarahnovotny
 - saschagrunert
 - SataQiu
+- sattarfeizollahibarough
```
Comment (spiffxp) [inline config/kubernetes/org.yaml:888]:
```suggestion
```

**Review by spiffxp — CHANGES_REQUESTED** [review]
Needs rebase: these members are already in the org

**Comment by spiffxp** [comment]
(this otherwise lgtm, please ping when resolved)

**Comment by sftim** [comment]
Aside: the related PRs and issues for adding Persian seem to be turning stale.

**Comment by nikhita** [comment]
Closing, please reopen if needed!

**Pull request closed without merging**